test/util/a11y.js

```diff
@@ -8,6 +8,8 @@ const pa11yRun = pa11y({
 });
 
 module.exports = (testPage, title) => {
+    console.log(`Test testPage being tested = ${testPage}`);
+    console.log(`Test title being tested = ${title}`);
```

Comment:
Pa11y already provides some output when running the tests, do we need these here as well?